src/clients/lan_auto.rb

```diff
@@ -159,7 +160,13 @@ def main
         @ret = deep_copy(@autoyast)
       elsif @func == "Write"
         @progress_orig = Progress.set(false)
-        @ret = Lan.WriteOnly
+
+        @ret = LanUdevAuto.Write
+        Builtins.y2error("Writing udev rules failed") if !@ret
+
+        @ret = Lan.WriteOnly && @ret
+        Builtins.y2error("Writing lan config failed") if !@ret
```

Comment:
so it writes `"Writing lan config failed"` even if only UdevAuto failed, is it correct?